nickspaargaren/sintinhuizen.nl
# Add photos of last year (2023)

### Context

Adds photos of last year (2023)

### How to test

1. Navigate to the previewapp
2. Verify that the 2023 photos are added
